content/post/deeplearning_in_go_part_1.md

```diff
@@ -0,0 +1,393 @@
+</g>
+<!-- fn&#45;&gt;fnFn -->
+<g id="edge7" class="edge"><title>fn&#45;&gt;fnFn</title>
+<path fill="none" stroke="black" d="M303.942,-143.697C303.942,-135.983 303.942,-126.712 303.942,-118.112"/>
+<polygon fill="black" stroke="black" points="307.442,-118.104 303.942,-108.104 300.442,-118.104 307.442,-118.104"/>
+</g>
+<!-- args0 -->
+<g id="node10" class="node"><title>args0</title>
+<ellipse fill="none" stroke="black" cx="587.942" cy="-90" rx="165.971" ry="18"/>
+<text text-anchor="middle" x="587.942" y="-86.3" font-family="Times,serif" font-size="14.00">*ast.BasicLit (Kind: STRING) (Value: %v)</text>
+</g>
+<!-- args&#45;&gt;args0 -->
+<g id="edge9" class="edge"><title>args&#45;&gt;args0</title>
+<path fill="none" stroke="black" d="M587.942,-143.697C587.942,-135.983 587.942,-126.712 587.942,-118.112"/>
+<polygon fill="black" stroke="black" points="591.442,-118.104 587.942,-108.104 584.442,-118.104 591.442,-118.104"/>
+</g>
+<!-- args1 -->
+<g id="node11" class="node"><title>args1</title>
+<ellipse fill="none" stroke="black" cx="868.942" cy="-90" rx="97.4827" ry="18"/>
+<text text-anchor="middle" x="868.942" y="-86.3" font-family="Times,serif" font-size="14.00">*ast.BinaryExpr (Op: +)</text>
+</g>
+<!-- args&#45;&gt;args1 -->
+<g id="edge10" class="edge"><title>args&#45;&gt;args1</title>
+<path fill="none" stroke="black" d="M625.839,-151.559C671.25,-140.247 748.104,-121.102 803.251,-107.364"/>
+<polygon fill="black" stroke="black" points="804.426,-110.679 813.284,-104.865 802.734,-103.886 804.426,-110.679"/>
+</g>
+<!-- lhs -->
+<g id="node12" class="node"><title>lhs</title>
+<ellipse fill="none" stroke="black" cx="718.942" cy="-18" rx="141.075" ry="18"/>
+<text text-anchor="middle" x="718.942" y="-14.3" font-family="Times,serif" font-size="14.00">*ast.BasicLit (Kind: INT) (Value: 1)</text>
+</g>
+<!-- args1&#45;&gt;lhs -->
+<g id="edge11" class="edge"><title>args1&#45;&gt;lhs</title>
+<path fill="none" stroke="black" d="M834.904,-73.1159C813.785,-63.26 786.413,-50.4867 763.567,-39.825"/>
+<polygon fill="black" stroke="black" points="764.951,-36.6089 754.409,-35.5516 761.991,-42.9522 764.951,-36.6089"/>
+</g>
+<!-- rhs -->
+<g id="node13" class="node"><title>rhs</title>
+<ellipse fill="none" stroke="black" cx="1018.94" cy="-18" rx="141.075" ry="18"/>
+<text text-anchor="middle" x="1018.94" y="-14.3" font-family="Times,serif" font-size="14.00">*ast.BasicLit (Kind: INT) (Value: 1)</text>
+</g>
+<!-- args1&#45;&gt;rhs -->
+<g id="edge12" class="edge"><title>args1&#45;&gt;rhs</title>
+<path fill="none" stroke="black" d="M902.979,-73.1159C924.099,-63.26 951.47,-50.4867 974.317,-39.825"/>
+<polygon fill="black" stroke="black" points="975.892,-42.9522 983.474,-35.5516 972.932,-36.6089 975.892,-42.9522"/>
+</g>
+</g>
+</svg>
+</div>
+
+By this, we can also say that any equation that can be represented as a computer program can be represented as a graph. In particular, let's zoom in `1+1` part:
```

Comment:
"... as a computer program, and a computer program can be represented by a graph."